olms.js

```diff
@@ -1,9 +1,9 @@
-import {apply, applyBackground, applyStyle} from './index';
+import olms, {apply, applyBackground, applyStyle} from './index';
 import stylefunction from './stylefunction';
 
-export default {
-  apply: apply,
-  applyBackground: applyBackground,
-  applyStyle: applyStyle,
-  stylefunction: stylefunction
-};
+olms.apply = apply;
+olms.applyBackground = applyBackground,
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove use of all comma operators in this expression. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sq.boundlessgeo.io/coding_rules#rule_key=javascript%3AS878)